Mathlib/Lean/Linter.lean

```diff
@@ -0,0 +1,20 @@
+/-
+Copyright (c) 2023 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import Lean
+
+/-!
+# Additional declarations related to linters
+-/
+
+open Lean Elab
+
+namespace Lean.Linter
+
+def logLint? [Monad m] [MonadLog m] [AddMessageContext m] [MonadOptions m]
```

Comment:
```suggestion
def logLintIf [Monad m] [MonadLog m] [AddMessageContext m] [MonadOptions m]
```
The `?` suffix is typically used to indicate an `Option` in the return type.

Reply:
You're right, this is different